tests/unittests/BackendTestUtils.cpp

```diff
@@ -432,6 +432,39 @@ void inferSmallConv(Tensor *inputs, Tensor *out, BackendKind kind) {
   out->copyFrom(&result->getVariable()->getPayload());
 }
 
+void inferGroupConv(Tensor *out, BackendKind kind) {
+  ExecutionEngine EE(kind);
+  auto &mod = EE.getModule();
+  auto *F = mod.createFunction("main");
+
+  auto *input = mod.createVariable(ElemKind::FloatTy, {1, 2, 1, 32}, "inp\
```

Comment:
I'm a bit confused about "inp\
Could you name it "input", and move the whole word to the next line? Or have clang-format move it for you.